fastlane/Fastfile

```diff
@@ -548,100 +541,97 @@ GH_ORG_NAME="woocommerce"
   lane :create_screenshot_summary do
     fastlane_require 'snapshot'
 
     # Provide enough information to bootstrap the configuration and generate the HTML report
     Snapshot.config = FastlaneCore::Configuration.create(Snapshot::Options.available_options, {
-      "workspace": "../WooCommerce.xcworkspace",
-      "scheme": "WooCommerceScreenshots"
-    })
+                                                           "workspace": '../WooCommerce.xcworkspace',
+                                                           "scheme": 'WooCommerceScreenshots'
+                                                         })
 
     Snapshot::ReportsGenerator.new.generate
   end
 
-  desc "Download App Store Translations"
-  lane :download_app_store_translations do | options |
-
+  desc 'Download App Store Translations'
+  lane :download_app_store_translations do |_options|
     files = {
 
       # Metadata fields
-      "app_store_desc"            => {desc: "description.txt"},
-      "app_store_keywords"        => {desc: "keywords.txt"},
-      "app_store_subtitle"        => {desc: "subtitle.txt"},
-      "app_store_promo_text"      => {desc: "promotional_text.txt"},
-      "whats_new"                 => {desc: "whats_new.txt"},
+      'app_store_desc' => { desc: 'description.txt' },
+      'app_store_keywords' => { desc: 'keywords.txt' },
+      'app_store_subtitle' => { desc: 'subtitle.txt' },
+      'app_store_promo_text' => { desc: 'promotional_text.txt' },
+      'whats_new' => { desc: 'whats_new.txt' },
 
       # Screenshots
-      "app_store_screenshot-1"    => {desc: "app_store_screenshot_1.txt"},
-      "app_store_screenshot-2"    => {desc: "app_store_screenshot_2.txt"},
-      "app_store_screenshot-3"    => {desc: "app_store_screenshot_3.txt"},
-      "app_store_screenshot-4"    => {desc: "app_store_screenshot_4.txt"},
-      "app_store_screenshot-5"    => {desc: "app_store_screenshot_5.txt"},
+      'app_store_screenshot-1' => { desc: 'app_store_screenshot_1.txt' },
+      'app_store_screenshot-2' => { desc: 'app_store_screenshot_2.txt' },
+      'app_store_screenshot-3' => { desc: 'app_store_screenshot_3.txt' },
+      'app_store_screenshot-4' => { desc: 'app_store_screenshot_4.txt' },
+      'app_store_screenshot-5' => { desc: 'app_store_screenshot_5.txt' }
     }
 
     metadata_locales = glotpress_locales
 
-    gp_downloadmetadata(project_url: "https://translate.wordpress.com/projects/woocommerce%2Fwoocommerce-ios%2Frelease-notes/",
-      target_files: files,
-      locales: metadata_locales,
-      source_locale: "en-US",
-      download_path: "./fastlane/metadata"
-    )
+    gp_downloadmetadata(project_url: 'https://translate.wordpress.com/projects/woocommerce%2Fwoocommerce-ios%2Frelease-notes/',
+                        target_files: files,
+                        locales: metadata_locales,
+                        source_locale: 'en-US',
+                        download_path: './fastlane/metadata')
```

Comment:
Is the Ruby standard to organize the code like this, or is this just a result of the first param being "inline" with the method name, do you know?

I find this better to read

```suggestion
    gp_downloadmetadata(
      project_url: 'https://translate.wordpress.com/projects/woocommerce%2Fwoocommerce-ios%2Frelease-notes/',
      target_files: files,
      locales: metadata_locales,
      source_locale: 'en-US',
      download_path: './fastlane/metadata'
    )
```

Rationale: methods with names of different length. will indent in the same way:

```ruby
short(a: 1,
      b: 2)

long_long_long_long(a: 1,
                    b: 2)
```

vs 

```ruby
short(
  a: 1,
  b: 2
)

long_long_long_long(
  a: 1,
  b: 2
)
```

Reply:
I agree I prefer your way too.

I think/hope this is mostly to align how the existing first line was indented initially; what happens if you fix the indentation manually and try to rerun rubocop?

Reply:
I sent the first argument on a new line then run `rubocop -a`.

```diff
     gp_downloadmetadata(
       project_url: 'https://translate.wordpress.com/projects/woocommerce%2Fwoocommerce-ios%2Frelease-notes/',
-                        target_files: files,
-                        locales: metadata_locales,
-                        source_locale: 'en-US',
-                        download_path: './fastlane/metadata')
+      target_files: files,
+      locales: metadata_locales,
+      source_locale: 'en-US',
+      download_path: './fastlane/metadata'
+    )
```

So, yeah, RuboCop fixes it in the way we both like. It also moved the closing `)` to a new line by itself, which was nice.

Reply:
🎉 👌

Reply:
This is way better, thanks!!